cli/studio.py

```diff
@@ -53,9 +64,17 @@ def _build_single_scenario(clean, scenario):
         _clean(scenario)
 
     scenario_root = Path(scenario)
-    map_net = scenario_root / "map.net.xml"
+    map_net = str(scenario_root / "map.net.xml")
+    if not allow_offset_map:
+        SumoRoadNetwork.from_file(map_net, shift_to_origin=True)
+    elif os.path.isfile(SumoRoadNetwork.shifted_net_file_path(map_net)):
```

Comment:
I guess this warning is to warn if user set `allow_offset_map` to False but there is an offset_map. 
If so, 
```suggestion
    elif not allow_offset_map and os.path.isfile(SumoRoadNetwork.shifted_net_file_path(map_net)):
```